scripts/e2e-android-ci.sh

```diff
@@ -16,7 +16,7 @@ echo "Running on devices: $DEVICES_LIST"
 # Install the app
 for DEVICE in $DEVICES; do
   echo "Install app on $DEVICE"
-  $ANDROID_HOME/platform-tools/adb -s "$DEVICE" install -r ./android/app/build/outputs/apk/release/app-release.apk &
+  adb -s "$DEVICE" install -r ./android/app/build/outputs/apk/release/app-release.apk &
```

Comment:
Not related, but I noticed we have this, but adb should be in path since we invoke it directly in a bunch of other places in the file.